### PR TITLE
Fix/ui fixes 0106

### LINKED ISSUE #717: [Multi Languages] To add the following for translation in airtable
The following are text which requires translation as they have been missed out.

Screenshot 2020-01-06 at 6 15 52 PM
Screenshot 2020-01-06 at 6 15 13 PM
Screenshot 2020-01-06 at 6 13 52 PM
Screenshot 2020-01-06 at 6 23 37 PM
Screenshot 2020-01-06 at 6 23 57 PM
Screenshot 2020-01-06 at 6 24 23 PM
Screenshot 2020-01-06 at 6 24 38 PM
Screenshot 2020-01-06 at 6 25 18 PM
Screenshot 2020-01-06 at 6 25 36 PM
Screenshot 2020-01-06 at 6 25 58 PM

### PR DESCRIPTION
**Pull request by lionellbriones**
- Updated popup skeleton placeholder
- Fixed missed translations
- Updated enum translations
Fixed #717

### CODE REVIEW

File: app/src/containers/WalletTransfer/WalletTransfer.vue [hunk 1]
```diff
@@ -394,6 +395,13 @@ export default {
     }
   },
   computed: {
+    verifierOptions() {
+      const verifiers = JSON.parse(JSON.stringify(ALLOWED_VERIFIERS))
```
Comment (chaitanyapotti) [inline app/src/containers/WalletTransfer/WalletTransfer.vue]: Why stringify and parse.?

File: app/src/containers/WalletHistory/WalletHistory.vue [hunk 2]
```diff
@@ -146,7 +184,7 @@ export default {
         } else if (activity.type === CONTRACT_TYPE_ERC20) {
           return `logos/${activity.type_image_link}`
         } else {
-          return `$vuetify.icons.coins_${activity.action.toLowerCase()}`
+          return `$vuetify.icons.coins_${activity.action.split('.')[1].toLowerCase()}`
```
Comment (chaitanyapotti) [inline app/src/containers/WalletHistory/WalletHistory.vue]: Check if split has length >=1

File: app/src/components/WalletSettings/ContactList/ContactList.vue [hunk 3]
```diff
@@ -102,18 +102,25 @@ export default {
...
-      verifierOptions: ALLOWED_VERIFIERS,
       ETH,
       saveContactAlert: false,
       saveContactAlertText: '',
       saveContactAlertType: 'success'
     }
   },
   computed: {
+    verifierOptions() {
+      const verifiers = JSON.parse(JSON.stringify(ALLOWED_VERIFIERS))
```
Comment (chaitanyapotti) [inline app/src/components/WalletSettings/ContactList/ContactList.vue]: Same as below

File: app/src/containers/WalletHistory/WalletHistory.vue (hunk 2 above)
Comment (lionellbriones) [reply app/src/containers/WalletHistory/WalletHistory.vue]: Done

File: app/src/containers/WalletTransfer/WalletTransfer.vue (hunk 1 above)
Comment (lionellbriones) [reply app/src/containers/WalletTransfer/WalletTransfer.vue]: This is for deep clone.. Cause i need to modify verifier labels with translation without affecting enums value.

File: app/src/components/WalletSettings/ContactList/ContactList.vue (hunk 3 above)
Comment (lionellbriones) [reply app/src/components/WalletSettings/ContactList/ContactList.vue]: This is for deep clone.. Cause i need to modify verifier labels with translation without affecting enums value.

**Pull request merged**